src/app/map/components/MapContainer.js

```diff
@@ -98,6 +100,30 @@ async function getData() {
         console.log(error.response.headers);
       }
     });
+
+  await axios.post(
+    process.env.REACT_APP_CORS + process.env.REACT_APP_API_ORGANIZATION_INFO,
+    { month: 8 },
```

Comment:
Consider using getMonth method to get current month

https://www.w3schools.com/jsref/jsref_getmonth.asp